src/components/StoreDisplay/ReviewList/ReviewList.tsx

```diff
@@ -8,40 +8,53 @@ import {
   ReviewEditButton,
 } from './ReviewList.styles'
 import { useSelector } from 'react-redux'
-import { RootState } from '@stores/store'
+import { RootState, useAppDispatch } from '@stores/store'
+import { deleteReview, fetchAllReviews } from '@stores/review/reivewSlice'
+import { useParams } from 'react-router-dom'
 
 interface ReviewType {
+  reviewId: number
   reviewContent: string
-  createdDate: Date
+  createdDate: string
   starCount: number
   keywords: string[]
   userId: string
 }
 
 const ReviewList: React.FC<ReviewType> = ({
+  reviewId,
   createdDate,
   reviewContent,
   starCount,
   keywords,
   userId,
 }) => {
-  const user = useSelector((state: RootState) => state.user.user)
+  const { storeId } = useParams()
   const [isWideView, setIsWideView] = useState<boolean>(false)
-  // createdDate 가 UTC 기준으로 들어올 경우 값을 조정해야함
-  const date = createdDate.toISOString().split('T')[0]
-
+  const dispatch = useAppDispatch()
+  const user = useSelector((state: RootState) => state.user.user)
+  const date = createdDate.split('T')[0]
+  const displayName = userId.split('@')[0]

```

Comment:
(p4) array destructuring 을 고려해보는건 어떨까요?
```suggestion
  const [date] = createdDate.split('T')
  const [displayName] = userId.split('@')
```

Reply:
const firstDir = createdDate.split('/')[0]
  const secondDir = createdDate.split('/')[1]
  const others = createdDate.split('/').slice(2)

  const [firstDir, secondDir, ...others] = createdDate.split('/')